### ISSUE TITLE
Assets won't compile, unexepcted flag --external=

### ISSUE THREAD

**Issue by dev-cyprium** [phoenixframework/phoenix#4539]
### Environment

* Elixir version (elixir -v):
```
Erlang/OTP 24 [erts-12.0.2] [source] [64-bit] [smp:5:5] [ds:5:5:10] [async-threads:1] [jit]

Elixir 1.12.1 (compiled with Erlang/OTP 24)
```
* Phoenix version (mix deps):
```
      {:phoenix, "~> 1.6.1"},
```

* Operating system:
WSL2 / Windows 10

### Expected behavior
When I created a new project it said unexpected flag external. And the assets don't work providing this error message.
![image](https://user-images.githubusercontent.com/6002628/136624269-6adae79e-3bdd-483a-aedb-e82bc327ec38.png)

### Actual behavior
The assets should compile without errors.

### Solution
I changed the args to use:
```elixir
   args:
      ~w(js/app.js --bundle --target=es2016 --outdir=../priv/static/assets --external:/fonts/* --external:/images/*),
```

**Comment by mcrumm** [comment]
You are absolutely right, which means it is on me! 😬 🙈 

Fix incoming, thank you!

**Comment by mdoyle13** [comment]
This is my first foray into Phoenix/Elixir and so far its off to a great start. Thanks for being so responsive!

**Comment by mcrumm** [comment]
Closed in a3a470e
